traffic_ops/traffic_ops_golang/cachegroup/queueupdate.go

```diff
@@ -88,7 +88,7 @@ func QueueUpdates(w http.ResponseWriter, r *http.Request) {
 		CDN:            *reqObj.CDN,
 		CacheGroupID:   cgID,
 	})
-	api.CreateChangeLogRawTx(api.ApiChange, "Server updates "+reqObj.Action+"d for "+string(cgName), inf.User, inf.Tx.Tx)
+	api.CreateChangeLogRawTx(api.ApiChange, "CACHEGROUP: "+string(cgName)+", ID: "+strconv.FormatInt(cgID, 10)+", ACTION: "+strings.Title(reqObj.Action)+"d CacheGroup server updates", inf.User, inf.Tx.Tx)
```

Comment:
there's one piece of info missing here. it really should be

ACTION: "+strings.Title(reqObj.Action)+"d CacheGroup server updates for the foo CDN

Reply:
this is because cache groups span across cdns